dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/CaseNode.java

```diff
@@ -62,7 +65,9 @@ public CaseTree getTree() {
 
   @Override
   public String toString() {
-    return "case " + getCaseOperand() + ":";
+    return "case "
+        + getCaseOperands().stream().map(Object::toString).collect(Collectors.joining(", "))
```

Comment:
I would use [`StringsPlume.join`](https://plumelib.org/plume-util/api/org/plumelib/util/StringsPlume.html#join-java.lang.CharSequence-java.lang.Iterable-) which is more concise and slightly more efficient.